gatling-http/src/main/scala/io/gatling/http/action/ws/WsConnect.scala

```diff
@@ -20,6 +20,7 @@ import io.gatling.commons.util.Clock
 import io.gatling.commons.validation._
 import io.gatling.core.CoreComponents
 import io.gatling.core.action.{ Action, RequestAction }
+import io.gatling.core.session._
```

Comment:
Weird. You're importing `session._` while the line below imports `session.{ Expression, Session }`. If you need something else, please remove the second line that becomes useless.